content/facilities/va-facilities/release-notes/2024-07-14

```diff
@@ -0,0 +1 @@
+Version 0 of the Facilities API is deactivated and has been removed from VA Developer. We deactivated this API because version 1 was launched providing enriched data with better API responses and functionality. Consumers who used version 0 were notified to switch to our latest version.
```

Comment:
what about the v1 API responses makes it better than v0?

Reply:
Was trying to avoid just repeating what is in the Release Note when v1 was launched that will be right below this one stating v0 is deprecated. But to list a few

- More health services available
- Cleaner and more concise data presentation
- Expanded search parameters
- Improved API response times